src/components/Properties.astro

```diff
@@ -1,8 +1,49 @@
-<div class="my-6">
+---
+const { heading = 'Attributes' } = Astro.props;
+---
+
+<div data-attributes class="my-6">
+  <div class="flex justify-between items-center not-prose py-2">
+    <h3 class="scroll-mt-10 type-subtitle-1">{ heading }</h3>
+    <button data-expand-button class="text-content-tertiary type-overline">
```

Comment:
Change: Can we check what the hover states are for the button? I would expect there should be some.

Reply:
there aren't any in the design by the looks of it  - do you have a suggestion for what it could be?

Reply:
@AndyClifford ask the jesters to give you a design 😄